content/developer/tutorials/importable_modules.rst

```diff
@@ -943,6 +943,20 @@ You then need to include the file in the appropriate bundle in the manifest file
         },
     }
 
+You also need to add an xml record in a new estate_tour.xml file in the data folder so that your tour is displayed:
+
+.. code-block:: xml
+    
+    <?xml version="1.0" encoding="utf-8"?>
+    <odoo>
+        <record id="estate_tour" model="web_tour.tour">
+            <field name="name">estate_tour</field>
+            <field name="sequence">2</field>
```

Comment:
The sequence is defined here, and not in the js anymore, you can remove it there (and I think the position is not taken into account anymore in js as well)